Composer/packages/client/src/router.tsx

```diff
@@ -42,6 +43,11 @@ const BotCreationFlowRouter = React.lazy(() => import('./components/CreationFlow
 const BotCreationFlowRouterV2 = React.lazy(() => import('./components/CreationFlow/v2/CreationFlow'));
 const FormDialogPage = React.lazy(() => import('./pages/form-dialog/FormDialogPage'));
 
+const modalControlGroup = css`
+  border: 1px solid rgb(237, 235, 233);
```

Comment:
Is there a matching border color from Shared, CommunicationColors?

Reply:
Ive removed the border as it is not necessary